src/devtools/views/Components/KeyValue.css

```diff
@@ -26,3 +26,13 @@
   color: var(--color-dimmer);
   font-style: italic;
 }
+
+.Opener {
+  padding: 0.125rem;
+  margin-left: -1rem;
+}
+
+.Opener svg {
```

Comment:
I don't know if this is desirable, since we are using css modules, but I think maybe the only other way is to allow `ButtonIcon` to receive a className?

Reply:
I'd rather avoid descendant selectors since they're slow. The pattern we follow in other components (e.g. `Button`, `Icon`) is to accept an optional `className` prop. `ButtonIcon` just didn't have a need for one before.